src/WooCommerce.php

```diff
@@ -552,11 +563,20 @@ public static function woocommerce_display_item_meta($html, $item, $args) {
     }
 
     foreach ($product_data_set as $productData) {
-      $strings[] = '<strong class="wc-item-meta-label">' . $productData['name'] . ':</strong> ' . $productData['value'];
+      $string = '';
+      if (!empty($productData['name'])) {
+        $string .= '<strong class="wc-item-meta-label">' . $productData['name'] . ':</strong> ';
+      }
+      if (!empty($productData['value'])) {
+        $string .= $productData['value'];
+      }
+      if (!empty($string)) {
+        $strings[] = $string;
+      }
     }
 
     if ($strings) {
-      $html = $args['before'] . implode($args['separator'], $strings) . $args['after'] . $html;
+      $html = $args['before'] . implode($args['separator'], $strings) . $args['after'];
```

Comment:
What's contained in the original `$html`?  Wouldn't it contain output from other plugins that gets lost by removing it?

Reply:
The `$html` is coming from the [`wc_display_item_meta()`](https://github.com/netzstrategen/gaco/blob/master/wp-content/plugins/woocommerce/includes/wc-template-functions.php#L3018-L3047) function and basically calls the same code I do here (`get_formatted_meta_data()`) to create the html string. So this should not contain any other information.